src/scripts/bad_medgen_xrefs_grep_command.py

```diff
@@ -0,0 +1,40 @@
+"""From a list of bad MedGen xrefs, generate a grep command to remove them from Mondo.
+
+See also:
+- GH issue: https://github.com/monarch-initiative/mondo-ingest/issues/273
+"""
+from argparse import ArgumentParser
+
+
+def run(input_file: str, output_file: str, target: str):
+    """Create grep command"""
+    bad_xref_cuis = []
+    with open(input_file, "r") as file:
+        for line in file:
+            bad_xref_cuis.append(line.strip())
+
+    output_file2 = output_file.replace('.txt', '-grep-command.sh')
+    command_string = "grep -v "
+    for s in bad_xref_cuis:
+        command_string += f'-e "xref: UMLS:{s}" '
```

Comment:
In `grep -v`, changed `-e {ID}` to `-e "xref: UMLS:{ID}"`

Reply:
I think Nico has had a chance to review this, and I don't think there is anything of consequence here. Instructions and  implementation have changed slightly.